src/React.AspNet/JsxFileSystem.cs

```diff
@@ -197,12 +197,12 @@ public DateTimeOffset LastModified
 
 			public void WriteContent(byte[] content)
 			{
-				_fileInfo.WriteContent(content);
+                File.WriteAllBytes(_fileInfo.PhysicalPath, content);
```

Comment:
Please use tabs rather than spaces.

Also why does this need to be changed? What's wrong with the old version? Passing through to the inner `IFileInfo` should be sufficient.